### ISSUE TITLE
Problema de ejecución

### ISSUE THREAD

**Issue by Peagram** [getmanfred/mac#22]
Cuando en la descripcion de un puesto de trabajo (job) se añade más de un challenge con solo descripción, solo se muestra el primero en la web que se genera.
![imagen](https://user-images.githubusercontent.com/3935764/181210415-54298d00-bf7e-4b63-a83f-1adcbbf6fc65.png)
![imagen](https://user-images.githubusercontent.com/3935764/181210528-848da388-e8b1-4c15-9b95-5d7f2d5d3bd8.png)
![imagen](https://user-images.githubusercontent.com/3935764/181210563-2976e50f-d932-45f1-9a54-50a3f6454482.png)

**Comment by ydarias** [comment]
Buenas @Peagram 

¿Podrías añadir un poco de contexto a qué te refieres con "la web que se genera"?

Si nos das la serie de pasos que has seguido intentaremos depurar el problema.

**Comment by Peagram** [comment]
Buenas, me refiero mi perfil en la web de manfred https://www.getmanfred.com/es/profile-preview/timeline
de hecho tras la ultima actualizacion que he realizado, no se ha actualizado ninguna de las descripciones ultimas de los puestos de trabajo.
El JSON lo he validado contra el esquema que teneis en este repositorio y es correcto, 

Podeis comprobar mi fichero aqui https://github.com/Peagram/Manfred-Json/blob/edd3dde126a950e9a30aa1ac3212f4b0b3b7095d/CV/MAC.json

**Comment by ydarias** [comment]
Ok @Peagram creíamos que se trataba de un problema con el MAC.

Sí, estamos todavía diseñando el flujo y expectativas de la importación de datos a Manfred a través de un MAC. 

El MAC es un formato abierto y libre para el intercambio de información, en ningún caso atado al modelo de datos de Manfred. Lo cuál implica que no todos sus componentes tienen una representación exacta en la web de Manfred. Aún nos queda bastante labor en los adaptador que transforman el MAC en el modelo de Manfred y viceversa, pero sobretodo en las expectativas que se crean con la gente que quiere usar el MAC como formato de intercambio.

Seguimos trabajando en ello, y cualquier idea es bienvenida.

**Comment by Peagram** [comment]
ok, entiendo que el MAC contiene mas informacion que puede que la web de manfred no muestre, pero los ultimos cambios que he añadido ni si quiera se han ejecutado, entiendo que el comportamiento deberia ser actualizar la parte que tiene implementada y lo demás no, pero en este caso no lo hace.

**Comment by dbonillaf** [comment]
@Peagram ¿te refieres a que ni siquiera los campos que sí son soportados ya por Manfred se han actualizado con tus últimos cambios en GitHub?

**Comment by Peagram** [comment]
exacto, es lo que mas me ha extrañado, en el repositorio, tengo hecho un tag (release/1.0 ) por si quereis echar un ojo a lo que pasa con esa version de MAC.json, el problema aparecia al añadir challenges y competences en los puestos de trabajo, no se si puede ser un problema de memoria, en el correo no he recibido ningun error.

**Comment by ydarias** [comment]
Es bastante extraño que no se sincronice y no esté enviando el correo de error, pero puede ser que tengamos algún bug. Lo anotamos para trabajar sobre ello.

Muchas gracias por el aviso y la ayuda :-)

**Comment by Peagram** [comment]
De nada, si necesitais que haga alguna prueba más me decís.
